src/server/api/controllers/messages-controller.js

```diff
@@ -61,4 +61,4 @@ module.exports = {
   deleteMessage,
   createMessage,
   editMessage,
-};
\ No newline at end of file
+};
```

Comment:
Deepthi, this was exactly the thing i was talking about in class! Make sure you commit stuff like this that is not relevant to your story!!